lib/checkPullRequestJob.js

```diff
@@ -144,9 +144,10 @@ const getRegistryReminder = (newJobFiles, changedFiles) => {
  */
 const getJobNameString = (newJobFiles) => {
   let jobNameString = '';
-  const jobNameArray = newJobFiles.map((file) =>
-    getJobNameFromFileName(file.filename)
```

Comment:
One suggestion: This gets a bit confusing when both functions are defined as `getJobNameFromFileName` and `getNewJobsFromFile`. Define them as `getJobFileNameFromFilePath` and `getNewJobsNameFromFileContent`. Also, fix other variable names in file i.e. refer as `jobFileName` and `jobName`

Reply:
Actually, rethinking this, `getJobNameFromFileName` should be `getJobModuleName`. Since it just returns the module name.

Reply:
Done.

Reply:
One last nit: for registry reminder:

`please add the new job ' + fileText + ' to the '+ jobRegistryFile +` -> This should not be job file/files, just job/jobs. Please add the new job(s) to the registry file.

Reply:
Done.